ramls/titles.raml

```diff
@@ -44,3 +44,27 @@ resourceTypes:
       collection-item:
         exampleItem: !include acq-models/mod-orders-storage/examples/title_get.sample
         schema: title
+
+  /unlink/{id}:
+    uriParameters:
+      id:
+        description: The UUID of a Title
+        type: UUID
+    delete:
+      description: Unlink a title from an instance
+      queryParameters:
+        deleteHolding:
+          displayName: The associated holding should be removed
```

Comment:
I think this should be "`deleteHoldings`" and "holdings". Or to be clear, I think this parameter should be "`forceDeletePiecesHoldingsAndItems`" (defaulting to `false`), because they are deleted by default, unless connected to another title.

Reply:
Updated